clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/ops/servergroup/ResizeDcosServerGroupAtomicOperation.groovy

```diff
@@ -46,10 +46,7 @@ class ResizeDcosServerGroupAtomicOperation implements AtomicOperation<Void> {
 
     def app = new App()
 
-    app.with {
-      id = appId.toString()
-      instances = size
-    }
+    app.instances = description.targetSize
```

Comment:
Previously you were setting id on the app definition as well. Is it not necessary on the app definition then (I see the modifyApp API takes it in).